velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -1588,8 +1588,10 @@ Expected<DateTimeResult> DateTimeFormatter::parse(
     }
   }
 
-  // Ensure all input was consumed.
-  if (cur < end) {
+  // Ensure all input was consumed if type_ is not LENIENT_SIMPLE or
+  // STRICT_SIMPLE datetime formatter.
```

Comment:
Perhaps use `if type_ is not simple datetime formatter.` for simplicity.

Reply:
Thanks. Updated.